packages/lib/server-only/document/find-documents.ts

```diff
@@ -477,5 +440,43 @@ const findTeamDocumentsFilter = (
 
       return filter;
     })
+    .with(ExtendedDocumentStatus.BIN, () => {
+      const filters: Prisma.DocumentWhereInput[] = [
+        {
+          teamId: team.id,
+          deletedAt: {
+            gte: DateTime.now().minus({ days: 30 }).startOf('day').toJSDate(),
+          },
+        },
+      ];
+
+      if (teamEmail) {
```

Comment:
![CleanShot 2024-09-18 at 14 20 14](https://github.com/user-attachments/assets/c377f9ce-59e0-4944-8baf-d97f02cb3d70)

The left-hand side is the example account which is the owner of the `pit` team. The example user sent the `documenso` document from the `pit` team to another user (2nd pic).

When I delete the document from the team account, it's in both the "Completed" and "Bin" tabs although the counting doesn't work (see the 0). It also places the document from the 2nd account (the recipient) in the `BIN`. Should it do that?